tools/sigh.ts

```diff
@@ -65,10 +66,11 @@ const steps: {[index: string]: ((args?: string[]) => boolean)[]} = {
   languageServer: [peg, build, buildLS, webpackLS],
   peg: [peg, railroad],
   railroad: [railroad],
-  test: [peg, railroad, build, runTests],
-  testShells: [peg, railroad, build, webpack, devServerAsync, testWdioShells],
+  test: [peg, railroad, build, lint, tslint, runTests],
```

Comment:
@mykmartin may be sad about this, I suspect some folks use this to iterate on tests, do we need to add lint here and next line?

Reply:
I don't remember why I did this exactly, my recollection is that I didn't make semantic changes. In any case, I don't need to do this here (probably).

Reply:
I would definitely argue against having lint as part of the test run sequence.

Reply:
Ok, I think my issue was that I kept committing PRs only to have Travis, et al, complain about lint errors. I realized lint is not run in the commands I use day-to-day. I understand what you are saying about slowing down test-cycle, so I will instead move this to the `webpack` top-level command (not the same as the `webpack` internal operation).